src/runtime_src/core/pcie/driver/linux/xocl/xocl_drv.h

```diff
@@ -2187,11 +2187,11 @@ struct xocl_xgq_vmr_funcs {
 		const void __user *arg, uint64_t slot);
 	int (*xgq_check_firewall)(struct platform_device *pdev);
 	int (*xgq_clear_firewall)(struct platform_device *pdev);
-	int (*xgq_freq_scaling)(struct platform_device *pdev,
+	int (*xgq_clk_scaling)(struct platform_device *pdev,
```

Comment:
Hi @prapulkrishnamurthy please see if this name also can be renamed to xgq_clk_throttling() in this PR?
And, xgq_clk_scaling_by_topo -> xgq_clk_throttling_by_topo()?

Reply:
Hi @rajkumar-xilinx , we came up with changes keeping a specific convention in mind. And we use the name "throttling" only for read/config sequence and "scaling" for read.